scripts/test_verify_clean_set_receipt.py

```
@@ -693,23 +693,25 @@ def test_exact_core_package_license_closure_passes(self) -> None:
             source, package = self._package_fixture(Path(temporary))
-            (package / "Cargo.toml").write_text(
+            # write_bytes avoids Windows CRLF translation that triggers the LF
+            # line-ending guard before the license check can run.
+            (package / "Cargo.toml").write_bytes(
                 (
                     '[package]\nname = "labcolors-core"\n\n'
                     '[[bin]]\nname = "fixture"\npath = "src/main.rs"\n'
                     f'license = "{CORE_LICENSE_EXPRESSION}"\n'
-                ),
-                encoding="utf-8",
+                ).encode("utf-8"),
```

Ruff (0.16.2)

[warning] Unnecessary UTF-8 `encoding` argument to `encode`

Remove unnecessary `encoding` argument

(UP012)